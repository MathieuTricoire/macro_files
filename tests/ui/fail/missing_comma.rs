use macro_files::create;

fn main() {
    create!({
        "README.md": "# Title"
        ".gitignore": "/target"
    });
}
