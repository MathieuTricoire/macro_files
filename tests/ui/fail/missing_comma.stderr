error: no rules expected the token `".gitignore"`
 --> tests/ui/fail/missing_comma.rs:6:9
  |
5 |         "README.md": "# Title"
  |                               - help: missing comma here
6 |         ".gitignore": "/target"
  |         ^^^^^^^^^^^^ no rules expected this token in macro call
